src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java

```diff
@@ -120,6 +123,29 @@ public int getCurrentQuantityInWard(
 
 		return (int) (mainQuantity != null ? mainQuantity : 0.0);	
 	}
+	
+	/**
+	 * Gets the current quantity for the specified {@link Ward} and {@link Lot}.
+	 * @param ward - if {@code null} the quantity is counted for the whole hospital
+	 * @param lot - the {@link code}
```

Comment:
is `code` suppose to be `lot`?;   the symbol `code` cannot be resolved in this context.

Reply:
Yep! Fixed